engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/ema/BigNumberEMAOperator.java

```diff
@@ -76,9 +78,10 @@ public BigNumberEMAOperator(@NotNull final MatchPair pair,
         this.control = control;
         this.valueSource = valueSource;
 
-        alpha = BigDecimal.valueOf(Math.exp(-1.0 / (double) timeScaleUnits));
-        oneMinusAlpha =
-                timestampColumnName == null ? BigDecimal.ONE.subtract(alpha, control.bigValueContextOrDefault()) : null;
+        opAlpha = BigDecimal.valueOf(Math.exp(-1.0 / (double) timeScaleUnits));
+        opOneMinusAlpha =
+                timestampColumnName == null ? BigDecimal.ONE.subtract(opAlpha, control.bigValueContextOrDefault())
+                        : null;
```

Comment:
The math here exists at least 3 places now. Should we write two one-line helper methods?